buildscripts/templates/registry/java/weaver.yaml

```diff
@@ -0,0 +1,59 @@
+comment_formats:
```

Comment:
Where can I read more about this file's contents?

Reply:
https://github.com/open-telemetry/weaver/blob/main/docs/weaver-config.md

Reply:
And here https://github.com/open-telemetry/weaver/tree/main/crates/weaver_forge#comment-filter